src/components/Table.js

```diff
@@ -60,8 +60,8 @@ export default class Table {
 
     // make table header fixed
     if (this.options.header) {
-      if (this.options.header.type && this.options.header.type === 'fixed') {
-        this.header.attr('class', 'mt-header-fixed');
+      if (this.options.header.className) {
+        this.header.attr('class', `${this.options.header.className}`);
 
         // set custom top header space
         if (this.options.header.top) {
```

Comment:
the header.top should be outside of the condition statement `if (this.options.header.className) {`

Here's a simplied version:

```js
if (this.options.header) {
  if (this.options.header.className) {
    this.header.attr('class', `${this.options.header.className}`);
  }

  this.header.attr('style', `top:${this.options.header.top || '0'}px;`);
}
```